configs/body_2d_keypoint/topdown_heatmap/coco/td-hm_2xrsn50_8xb32-210e_coco-256x192.py

```diff
@@ -30,11 +30,15 @@
 default_hooks = dict(checkpoint=dict(save_best='coco/AP', rule='greater'))
 
 # codec settings
-codec = dict(
-    type='MegviiHeatmap',
-    input_size=(192, 256),
-    heatmap_size=(48, 64),
-    kernel_size=5)
+# multiple kernel_sizes of heatmap gaussian for 'Megvii' approach.
+kernel_sizes = [15, 11, 9, 7] + [11, 9, 7, 5]
```

Comment:
Why it's the sum of 2 lists here with duplicated kernel size?

Reply:
This is because the encoding is prepared for 2 stages, and the kernel_sizes are the same for the last 3 levels in the first stage and the first 3 levels for the second stage. So I seperate the 2 stages' kernel_sizes to make it clear that each list of kernel sizes is for a stage.

On the one hand, I think the encoding may be redundant. But on the other hand, we should also leave the space for the possible modifications in configs. Is there any good solution to balance these two worries?

Reply:
In this case, encoding with kernel sizes 7, 9, and 11 will be calculated twice which is unnecessary. I suggest user `kernel_sizes=[15, 11, 9, 7, 5]` and specify the level indices for different stages at the head configs.